File: existant/gestFormationWeb/gestion_formation_web/dto/DtoSalle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using gestion_formation_web.dao;

namespace gestion_formation_web.dto
{
    public class DtoSalle
    {
        public static salle Get(int idSalle)
        {
            return DaoSalle.Get(idSalle);
        }
    }
}

File: Gestionnaire de formation/src/main/java/fr/esic/solutec/dto/DtoSalle.java
package fr.esic.solutec.dto;

import java.util.List;


import fr.esic.solutec.dao.DaoSalle;
import fr.esic.solutec.entities.Salle;

public class DtoSalle {

	
	public static Salle getSalle(int idSalle) {
		
		return DaoSalle.getSalle(idSalle);
	}
	
	public static List<Salle> getListSalle() {
		
		return DaoSalle.getListSalle();
	}
	
	
	
	
}
